editor.planx.uk/src/pages/FlowEditor/lib/store/editor.ts

```diff
@@ -365,7 +365,6 @@ export const editorStore: StateCreator<
         toBefore,
       })(get().flow);
       send(ops);
-      get().resetPreview();
```

Comment:
so this is a notable change! `moveNode` is the only graph store method that called this shared store method, which causes the window to reload, the user to no longer be set, and therefore the flow incorrectly reloads as if it's now view-only rather than edit-access as intended.

from what I can tell, `resetPreview` is intended to be called outside the Editor based on the variables it's setting, and in testing I can't see any in-editor issues with removing it (the operation is still successful, the flow order after move persists as expected, etc). But please do test this!